node_modules/gh-series/tests/test-crud.js

```diff
@@ -178,22 +178,41 @@ describe('Series', function() {
         });
 
         /**
-         * Test that verifies authorization for a serie
+         * Test that verifies authorization for a series
          */
-        it('verify authorization when getting a serie', function(callback) {
-            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 1, false, function(simon) {
+        it('verify authorization when getting a series', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 2, false, function(simon, sam) {
                 TestsUtil.generateTestUsers(global.tests.apps.cam2014, 1, false, function(nico) {
 
-                    // Create a serie on the 2013 app
-                    SeriesTestUtil.assertCreateSerie(simon.client, 'Test serie', null, function(createdSerie) {
+                    // Create a series on the 2013 app
+                    SeriesTestUtil.assertCreateSerie(simon.client, 'Test series', null, function(createdSeries) {
 
                         // Assert that it can't be retrieved on the 2014 app
-                        SeriesTestUtil.assertGetSerieFails(nico.client, createdSerie.id, false, 401, function() {
+                        SeriesTestUtil.assertGetSerieFails(nico.client, createdSeries.id, false, 401, function() {
 
                             // Assert a global admin can retrieve it
                             TestsUtil.getGlobalAdminRestClient(function(adminClient) {
-                                SeriesTestUtil.assertGetSerie(adminClient, createdSerie.id, false, createdSerie, function(retrievedSerie) {
-                                    return callback();
+                                SeriesTestUtil.assertGetSerie(adminClient, createdSeries.id, false, createdSeries, function(series) {
+                                    assert.strictEqual(series.canManage, true);
```

Comment:
Might be worth making this check part of the test util, so we can at least test that it's always returned and a boolean